docs/source/reference/command_line.rst

```diff
@@ -348,7 +348,7 @@ Below is a list with all available subcommands.
     Options:
       --profile-name TEXT  Name of the profile. By default, a unique name starting with
                            `presto` is automatically generated.  [default: (dynamic)]
-      --email TEXT         Email of the default user.  [default: aiida@localhost]
+      --email TEXT         Email of the default user.  [default: alexander.goscinski@psi.ch]
```

Comment:
Need to revert this. This happens when you have the `autofill.email` option defined, which is automatically done when you run the `verdi quicksetup` command once. It is a bit unfortunate that this is automatically converted by the `verdi-autodocs` pre-commit hook. Will need to find a way somehow to ignore this change, but not sure how yet.

Reply:
I defended it several times successful from being pushed, but it eventually it got in.

Removed.